runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -978,12 +988,8 @@ public DataflowPipelineJob run(Pipeline pipeline) {
     dataflowOptions.setPipelineUrl(stagedPipeline.getLocation());
 
     if (!isNullOrEmpty(dataflowOptions.getDataflowWorkerJar())) {
```

Comment:
I think we should only stage dataflow worker jar when `! useUnifiedWorker ()`. I can have a cleanup PR for that after this one is in.